### ISSUE TITLE
setting telegram

### ISSUE THREAD

**Issue by KraineOpasen** [Tkd-Alex/Twitch-Channel-Points-Miner-v2#632]
Hello, can I make the same video as for discord on setting up telegram?

**Comment by AdamT20054** [comment]
> Hello, can I make the same video as for discord on setting up telegram?

?

If you want it to post to telegram too, go to the example.py in your local version and throw in the values needed in the Telegram part.

**Comment by KraineOpasen** [comment]
thanks for the answer, but I don't really understand how to get the id and bind the bots.  I read the comments here on setting up, but I never managed to set it up.  If someone could explain step by step how to set it up, that would be great.

**Comment by ColinShark** [comment]
> If someone could explain step by step how to set it up, that would be great.

It's in the Readme. https://github.com/Tkd-Alex/Twitch-Channel-Points-Miner-v2#telegram

**Comment by KraineOpasen** [comment]
> > If someone could explain step by step how to set it up, that would be great.
> 
> It's in the Readme. https://github.com/Tkd-Alex/Twitch-Channel-Points-Miner-v2#telegram
yeah, I read it, it was not possible to configure according to this scheme
